shotover-proxy/src/message/mod.rs

```diff
@@ -102,6 +102,14 @@ impl Message {
         }
     }
 
+    pub fn get_namespaced_primary_key(&self) -> Option<String> {
+        unimplemented!()
+    }
+
+    pub fn query_values(&self) -> Option<Vec<String>> {
+        unimplemented!()
+    }
+
```

Comment:
What will these be returning?

Reply:
I added some comments to explain what they did.